Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Base on offical Node.js Alpine image
+FROM node:current-alpine AS base
+
+# Set working directory
+WORKDIR /base
+
+# Copy package.json and package-lock.json before other files
+# Utilise Docker cache to save re-installing dependencies if unchanged
+COPY package*.json yarn.lock ./ 
+
+# Install dependencies
+RUN yarn install --frozen-lockfile
+
+# Copy all files
+COPY . .
+
+# Build app
+FROM base AS builder
+ENV NODE_ENV=production
+WORKDIR /build
+COPY . .
+COPY --from=base /base/node_modules ./node_modules
```

Comment:
If my understandings are correct, the above command is for copying everything in the `base` to `build`. Then why are you copying the `node_modules` again? 🤔

Reply:
I'll modify it to be `COPY` all files from `build stage` instead.